src/ApiPort/ApiPort.GUI/README.md

```diff
@@ -1,6 +1,6 @@
-## Portability Analyzer 
+## .NET Portability Analyzer 
 
-The Portability Analyzer is an open source standalone tool that simplifies your porting experience by identifying APIs that are not portable to .NET Core. We discovered that developers are struggling to port their desktop applications from .NET Framework to .NET Core, therefore in order to meet customer needs we updated the .NET Portability Analyzer. The Portability Analyzer is an open source tools that simplifies your porting experience by identifying APIs that are not portable among the various .NET Platforms. 
+The .NET Portability Analyzer is an open source standalone tool that simplifies your porting experience by identifying APIs that are not portable to .NET Core. We discovered that developers are struggling to port their desktop applications from .NET Framework to .NET Core, therefore in order to meet customer needs we updated the .NET Portability Analyzer. The Portability Analyzer is an open source tools that simplifies your porting experience by identifying APIs that are not portable among the various .NET Platforms. 
 
 Running the tool can do three things: 
 
```

Comment:
The below steps:
1. Only .NET Core 3.0? Or other versions/frameworks as well?
2. This sentence is a little confusing. "You can also have the tool generate a report you can export in ...."
3. "To determine API investments going forwards, Microsoft also collects the same data that is generated in the spreadsheet so we can evaluate implementing the most-used APIs. None of your..." 

Is there an opt-out for data collection?